nodes/viz/viewer_texture.py

```diff
@@ -370,12 +370,9 @@ def process(self):
         size_tex = 0
         width = 0
         height = 0
-        if self.color_mode in ('RGB', 'RGBA'):
-           cMode = ((True,))
-        else:
-           cMode = ((False,))
 
-        print(cMode)
+        # why (( )) ?
+        cMode = ((True,)) if self.color_mode in ('RGB', 'RGBA') else ((False,))
 
```

Comment:
because `uniform_bool` accept a sequence of bool [see](https://docs.blender.org/api/blender2.8/gpu.types.html?highlight=uniform_botiger_augmented_reality_abstract_art.jpgol#gpu.types.GPUShader.uniform_bool) at least i get an error trying with a single values. That's strange and didn't understood why the accept only seq.

Reply:
cool :)

Reply:
will remove comment or add add reason with link.